integration_tests/e2e/personal/edit/address/confirmAddress.cy.ts

```diff
@@ -0,0 +1,111 @@
+import { v4 as uuidv4 } from 'uuid'
+import Page from '../../../../pages/page'
+import { Role } from '../../../../../server/data/enums/role'
+import ConfirmAddressPage from '../../../../pages/editPages/address/confirmAddressPage'
+import { AddressRequestDto } from '../../../../../server/data/interfaces/personIntegrationApi/personIntegrationApiClient'
+import NotFoundPage from '../../../../pages/notFoundPage'
+
+context('Confirm Address Page', () => {
+  const prisonerNumber = 'G6123VU'
+  const bookingId = 1102484
+  const addressCacheId = uuidv4()
+  const addressKey = `ephemeral:${addressCacheId}`
+  const address: AddressRequestDto = {
+    buildingNumber: '1',
+    subBuildingName: 'Sub Building Name',
+    buildingName: 'Building Name',
+    thoroughfareName: 'My Street',
+    dependantLocality: 'My Locality',
+    postTownCode: 'CITY1',
+    countyCode: 'COUNTY1',
+    countryCode: 'COUNTRY1',
+    postCode: 'A1 2BC',
+    fromDate: '2025-06-10',
+    addressTypes: [],
+  }
+  const cachedAddress = JSON.stringify({ address, route: 'find-uk-address' })
+
+  before(() => {
+    // Seed redis data:
+    cy.exec(`./integration_tests/scripts/redis-cli "SET ${addressKey} '${cachedAddress.replaceAll('"', '\\"')}'"`)
```

Comment:
I'm not sure if this will ever be needed elsewhere so probably not necessary, but if it was i'd probably:

- Make this a cypress command so that it's clear what it's doing (maybe something like `cy.redisSet(key, value)`
- Make `cachedAddress.replaceAll(...)` a variable with a descriptive name so that it doesn't require parsing the regex if you don't need to